pmd-plsql/src/main/java/net/sourceforge/pmd/lang/plsql/ast/ASTFormalParameter.java

```diff
@@ -19,6 +23,30 @@ public ASTFormalParameter(PLSQLParser p, int id) {
         super(p, id);
     }
 
+    public boolean isIn() {
+        return this.in;
+    }
+
+    public void setIn(boolean in) {
```

Comment:
we should make all setters package private, so that it can only be called from the parser....